docstring_parser/epydoc.py

```diff
@@ -191,6 +191,16 @@ def parse(text: str) -> Docstring:
 
         ret.meta.append(meta_item)
 
+    # Add support for just specifying the return type without a description:
+    if not is_done.get("return", True):
+        info = params["return"]
+        meta_item = DocstringReturns(
+            args=[key],
+            description=info.get("description"),
+            type_name=info.get("type_name"),
+            is_generator=info.get("is_generator", False),
+        )
```

Comment:
We're not doing anything with `meta_item` and yet the code is working?
Also why do we default to `is_done.get("return")` to `True`?